### DISCUSSION TITLE
git pull

### DISCUSSION THREAD

**Discussion by crubik** [xtrime-ru/antizapret-vpn-docker#73] [Q&A] [answered]
после обновления ловлю такую ошибку
![Screenshot_1](https://github.com/user-attachments/assets/c74a079f-2baf-407f-a432-f83e02ff5849)
как-то можно исправить?

**Comment by xtrime-ru** [comment, marked as answer]
Чистил историю гита, поэтому конфликт. 

Это команда затрет все изменения в файлах и принудительно обновится до последнего комита: 
```
git fetch && git reset --hard origin/master

```
